### DISCUSSION TITLE
A new TiddlyWiki5 Highlight&Hint plugin for CodeMirror

### DISCUSSION THREAD

**Discussion by Gk0Wk** [TiddlyWiki/TiddlyWiki5#5990] [General]
Hello all,

I recently write a new plugin for tw5 to adds syntax Highlighting & Hint for TiddlyWiki5 tiddlers (`text/vnd.tiddlywiki`) to the CodeMirror.

If you need these functions in your wiki, you can have a try. The plugin is still under development, so please back up it before installing.

Download: https://github.com/Gk0Wk/CodeMirror-Mode-TiddlyWiki5

**Comment by kookma** [comment]
I may recommend to announce this plugin on https://talk.tiddlywiki.org/

**Reply by Gk0Wk** [reply]
> I may recommend to announce this plugin on https://talk.tiddlywiki.org/

Thanks a lot! I'm new to this website. Which tag should I add to the topic, the `Plugin` one?

Besides, I notice that my topic in Google Group has been synced to this site(https://talk.tiddlywiki.org/t/tw5-a-new-tiddlywiki5-highlight-hint-plugin-for-codemirror/398/2). Does this mean I don't need to double post topics?

**Comment by saqimtiaz** [comment]
Note that this plugin also includes link autocompletion. Nice work @Gk0Wk

**Reply by Gk0Wk** [reply]
Thank you! The auto-completion method is now temporarily based on [TW5-CodeMirror-Plus](https://github.com/adithya-badidey/TW5-codemirror-plus), which is also a good job.
